src/index.css

```diff
@@ -10,6 +10,9 @@ body {
   display: flex;
   flex-direction: column;
   align-items: center;
+  background-image: url(https://png.pngtree.com/background/20210714/original/pngtree-digital-technology-gaming-background-with-circuit-picture-image_1247010.jpg);
```

Comment:
- [ ] 👍 Thank you for your effort in adding the background image for the project! However, I would recommend making some adjustments to the image or consider removing it. The current background image might be impacting the readability of the scores and the form to add new scores due to its low contrast. To ensure a better user experience, you can either remove it and use white color as in the wireframe or choose a different background that allows the scores and form to be more visible and easily readable.

By making these adjustments, you'll improve the overall usability and aesthetics of your project. Keep up the great work! 🚀🎨